content/ja/docs/reference/glossary/userns.md

```diff
@@ -0,0 +1,24 @@
+---
+title: User Namespace
+id: userns
+date: 2021-07-13
+full_link: https://man7.org/linux/man-pages/man7/user_namespaces.7.html
+short_description: >
+  非特権ユーザーに対して管理者権限をエミュレートするLinuxカーネルの機能。
+
+aka:
```

Comment:
Optionally:
```suggestion
aka:
 - ユーザー名前空間
```

Reply:
or vice versa: translate it in the article and write aka: user namespace

Reply:
I wrote `aka: user namespace` as @nasa9084 suggested.
Commit ID: 61b7bac51b9b329280e137b8cf160f6a4ba29421

@sftim @nasa9084 
This will help Japanese readers understand what ユーザー名前空間 is in English. Thank you!